lib/edit-site-page.php

```diff
@@ -167,8 +167,8 @@ function gutenberg_edit_site_init( $hook ) {
 	$preload_paths = array(
 		'/',
 		'/wp/v2/types?context=edit',
-		'/wp/v2/taxonomies?per_page=100&context=edit',
-		'/wp/v2/pages?per_page=100&context=edit',
+		'/wp/v2/taxonomies?context=edit',
+		'/wp/v2/pages?context=edit',
```

Comment:
```suggestion
		'/wp/v2/pages?per_page=100&context=edit',
```
This should not be changed.

Reply:
Why?

Reply:
Because pages is paginated, as pages requires a database call. Where taxonomies is call to registered taxonomies and is not paginated. 

I will flip the question around on your @epiqueras why was it changed in the first place?

Reply:
Because we don't need 100 per page, the default is fine.

Reply:
It was never 100. It was always the default.

Reply:
Gotcha. LGTM!